components/bundlers/webpack-css-modules/readme.md

```diff
@@ -1,19 +1,34 @@
 # Webpack bundler
 
-A component [compiler](https://docs.bitsrc.io/docs/ext-compiling.html) that bundles the component using [webpack](https://webpack.js.org/) v4.1.1.
+A component [compiler](https://docs.bitsrc.io/docs/ext-compiling.html) that bundles the component using [webpack](https://webpack.js.org/) ^v4.0.1.
 
 ## What is bundled?
```

Comment:
I think we should also link to [this page](https://docs.bit.dev/docs/best-practices.html#prefer-transpiling-over-bundling) from the docs, as for components it's better to transpile.